src/docs/self-hosted/releases.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: 'Self-Hosted Versions & Releases'
+title: 'Self-Hosted Releases & Upgrading'
 ---
 
 ![CalVer: YY.MM.MICRO](https://img.shields.io/badge/calver-YY.MM.MICRO-ad6caa.svg)
```

Comment:
While we're here ... what is the value of this badge? It doesn't give our actual version, sooooo ...

Reply:
I mean it looks cool (😁) and it immediately communicates the versioning schema. Can we not keep it?